### ISSUE TITLE
does not support bool type?

### ISSUE THREAD

**Issue by ma91n** [ianlopshire/go-fixedwidth#39]
go-fixedwidth does not support bool type in Struct?

```go
package main

import (
	"fmt"
	"github.com/ianlopshire/go-fixedwidth"
	"log"
	"strings"
)

func main() {

	type Book struct {
		F1 bool `fixed:"1,5,left, "`
		F2 bool `fixed:"6,10,left, "`
		F3 int  `fixed:"11,12"`
	}

	s := `true true 1
falsefalse2
true false3`

	for _, s := range strings.Split(s, "\n") {
		var b Book
		if err := fixedwidth.Unmarshal([]byte(s), &b); err != nil {
			log.Fatal(err)
		}

		fmt.Printf("%+v\n", b)
	}

}
```

Then output ix below.

```
2021/01/31 23:23:16 fixedwidth: cannot unmarshal true true 1 into Go struct field Book.F1 of type bool:fixedwidth: unknown type
exit status 1
```

I want support for bool types.

**Comment by ianlopshire** [comment]
@ma91n, You are correct. The library doesn't currently have support for the `bool` type. There aren't any current plans to add support but a PR would be welcome.

If you are interested, adding support shouldn’t be a major undertaking. Here are the best places to start to add encoding and decoding support respectively:

https://github.com/ianlopshire/go-fixedwidth/blob/54451b9969a30ffd4808b525ca796432d3b1f333/encode.go#L134

https://github.com/ianlopshire/go-fixedwidth/blob/54451b9969a30ffd4808b525ca796432d3b1f333/decode.go#L285

**Comment by ianlopshire** [comment]
Resolved by #42
